api-tests/features/disabled-cri-journeys.feature

```diff
@@ -0,0 +1,233 @@
+      Then I get a 'pyi-another-way' page response
+
+    Scenario: Separate session mitigation with enhanced verification and access-denied from DCMAW leads to ineligible
+      Given the subject already has the following credentials
+        | CRI        | scenario                            |
+        | ukPassport | kenneth-passport-valid              |
+        | address    | kenneth-current                     |
+        | fraud      | kenneth-score-2                     |
+        | kbv        | kenneth-needs-enhanced-verification |
+      And I activate the 'f2fDisabled' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit an 'access-denied' event
+      Then I get a 'pyi-another-way' page response
+
+  Rule: TICF is disabled
+
+    Scenario: A P2 journey is still successful
+      Given I activate the 'ticfDisabled' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub
+      Then I get a 'page-dcmaw-success' page response
+      When I submit a 'next' event
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity without a TICF VC
+
+  Rule: BAV is disabled
+
+    Scenario: Not having ID suitable for F2F leads to the escape page
+      Given I activate the 'bavDisabled' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'end' event
+      Then I get a 'page-ipv-identity-postoffice-start' page response
+      When I submit an 'end' event
+      Then I get a 'pyi-escape' page response
+
+    Scenario: Choosing to prove your identity another way on web journey leads to the escape page
+      Given I activate the 'bavDisabled' feature set
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit an 'access-denied' event
+      Then I get a 'page-multiple-doc-check' page response
+      When I submit an 'end' event
+      Then I get a 'pyi-post-office' page response
+      When I submit an 'end' event
+      Then I get a 'pyi-escape' page response
+
+  Rule: HMRC and DWP KBVs are disabled
```

Comment:
Should there be another case for 'DWP KBVs are disabled' (but HMRC KBVs are enabled)?

Feels like this might overlap with some of the other work that Hilux are doing, if so then we could potentially put them in a separate suite. (cc @blakeyp)